cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -0,0 +1,115 @@
+
+{# ==========================================================================
+
+   post_preview.render()
+
+   ==========================================================================
+
+   Description:
+
+   Render an article when given:
+
+   settings:
+
+   settings.post_title:         A string with the title of the post.
```

Comment:
Should these be `settings.heading` and `settings.body` to make them align w/ other similar macros?